0006-actor-to-actor.md

```diff
@@ -0,0 +1,62 @@
+# Actor-to-Actor Calls Allowed by Default
+
+In this decision record we discuss the choice of allowing actors to call operations on other actors
+by default.
+
+## Context and Problem Statement
+
+wasmCloud is a _zero trust_ security model, and so when people look at the default security policies they want
+to know whether it is appropriate for actors to be allowed to call each other by default. For comparison, actors
+cannot interact with capability providers without having an established link definition between them created at
+runtime.
+
+The rest of this ADR discusses the options and decisions, as well as the tradeoffs involved, in determining whether
+or not actors should be allowed to communicate directly by default.
+
+## Considered Options
+
+* Allow by Default
+* Deny by Default
+* Create Explicit Actor Calling Policies
+
+### Allow by Default
+
+This option essentially provides no _built-in_ rules preventing communication between actors. Developers and administrators can still use the [authorizer](https://wasmcloud.dev/reference/custom/authorizers/) trait facility of the wasmCloud host to create their own policies, but wasmCloud opts out of this decision-making process.
```

Comment:
This would be a great sample authorizer to provide, where an actor-to-actor call can only be to an actor with a subset (equal sets allowed) of the same capability claims.